src/logger/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .Logger import Logger
+from .Logger import Severity
```

Comment:
This is cool and I'm stoked to have a working example of this to learn from, thanks!